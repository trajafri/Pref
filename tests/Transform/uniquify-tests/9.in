(define x (lambda (x) (let ([x (+ 3 x)]) x)))

(define y (lambda (x) (let ([x (+ 3 x)]) z)))

(define x 3)

(lambda (z) (let ([z (+ 3 x)]) z))
