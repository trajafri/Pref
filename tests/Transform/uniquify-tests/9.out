(dfine x.0 (lambda (x.1) (let ([x.2 (+ 3 x.1)]) x.2)))

(define y.0 (lambda (x.3) (let ([x.4 (+ 3 x.3)]) z.-1)))

(define x.5 3)

(lambda (z.0) (let ([z.1 (+ 3 x.5)]) z.1))
